src/popup.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Serenade for Chrome</title>
+    <link rel="stylesheet" href="popup.css" />
+  </head>
+  <body>
+    <h1>Serenade for Chrome</h1>
+    <a href="https://serenade.ai/docs/chrome/"> Read the docs </a>
+    <br />
+    <label class="switch">
+      <input id="showClickables" type="checkbox" />
+      <span class="slider round"></span>
+    </label>
+    Always show clickables
```

Comment:
super minor thing—let's call this something like "link overlays" in the UI, since "clickable" is a bit confusing.